packages/app-builder/package.json

```diff
@@ -21,19 +21,25 @@
     "commit": "concurrently \"yarn test --coverage --no-cache --silent --forceExit --detectOpenHandles --runInBand --watch=false && jest-coverage-badges --input src/tests/coverage/coverage-summary.json --output src/tests/badges\" \"yarn lint\""
   },
   "dependencies": {
+    "@craftjs/core": "^0.1.0-beta.19",
+    "@craftjs/layers": "^0.1.0-beta.19",
     "@linaria/core": "^3.0.0-beta.4",
     "@linaria/react": "^3.0.0-beta.7",
+    "@material-ui/core": "^4.12.0",
     "@reapit/connect-session": "workspace:packages/connect-session",
     "@reapit/elements": "workspace:packages/elements",
     "@reapit/elements-legacy": "workspace:packages/elements-legacy",
     "@sentry/browser": "^6.9.0",
     "dayjs": "^1.10.6",
     "react": "^17.0.2",
+    "react-contenteditable": "^3.3.5",
     "react-dom": "^17.0.2",
+    "react-frame-component": "^5.1.0",
     "react-ga": "^2.7.0",
     "react-router": "^5.2.0",
     "react-router-dom": "^5.2.0",
     "snyk": "^1.662.0",
+    "styled-components": "^5.3.0",
```

Comment:
So we already have linaria which gives you styled tags in the project - would prefer not to support two CSS in JS solutions unless there is a persuasive reason